source/core_mqtt_serializer.c

```diff
@@ -934,7 +934,7 @@ static MQTTStatus_t deserializeConnack( const MQTTPacketInfo_t * pConnack,
      * "Remaining length" of 2. */
     if( pConnack->remainingLength != MQTT_PACKET_CONNACK_REMAINING_LENGTH )
     {
-        LogError( ( "CONNACK does not have remaining length of %d.",
+        LogError( ( "CONNACK does not have remaining length of %u.",
                     MQTT_PACKET_CONNACK_REMAINING_LENGTH ) );
```

Comment:
I believe this is defined is a `uint8_t`, so would it need to be cast as well?

Reply:
Yes, you are right. Did not notice the macro had a cast in it as well. Thank you for catching 👍🏽